src/toTransform.js

```diff
@@ -0,0 +1,151 @@
+(function () {
+  function InvalidArgument (message) {
+    this.message = message;
+    this.name = 'InvalidArgument';
+  }
+
+  function isNumeric (number) {
+    return !isNaN(number);
+  }
+
+  function convertTranslate (properties) {
+    var result = '';
+    if (!(properties.translate !== undefined && properties.translate !== 'none')) {
+      return result;
+    }
+    result += 'translate(';
+
+    var values = properties.translate;
+    var valuesArray = values.split(' ');
+    var numValues = valuesArray.length;
+
+    if (numValues > 3) {
+      throw new InvalidArgument('Too many values for translate');
+    }
+
+    if (numValues === 1 && valuesArray[0] === '') {
+      throw new InvalidArgument('Empty string is not a valid argument');
+    }
+
+    for (var i = 0; i < numValues; i++) {
+      var elementLength = valuesArray[i].length;
+      if (elementLength <= 2) {
+        throw new InvalidArgument('Incorrect argument for translate');
+      }
+
+      // Check if units are px
+      var unit = valuesArray[i].substring(elementLength - 2, elementLength);
+      if (unit !== 'px') {
+        throw new InvalidArgument('Incorrect units for translate');
+      }
+
+      // Check if characters before 'px' are valid numbers
+      var number = valuesArray[i].substring(0, numValues - 2);
+      if (!isNumeric(number)) {
+        throw new InvalidArgument('Argument must be a number');
+      }
+    }
+
+    result += valuesArray.join(', ') + ')';
+
+    return result;
+  }
+
+  /**
+  * This function is a WIP.
+  */
```

Comment:
Let's not check in code that's not ready, remove it from this patch, you can always retrieve it later from your branch's commits.